databases.go

```diff
@@ -299,6 +300,14 @@ type DatabaseCreatePoolRequest struct {
 	Mode     string `json:"mode"`
 }
 
+// DatabaseUpdatePoolRequest is used to update a database connection pool
+type DatabaseUpdatePoolRequest struct {
+	User     string `json:"user"`
```

Comment:
are any of the struct vales within the update request required?

Reply:
Yes, `size`, `database`, and `mode`.